src/app/shared/forms/planet-tag-input-dialog.component.html

```diff
@@ -2,7 +2,7 @@
   <span mat-dialog-title i18n *ngSwitchCase="'filter'">Collections</span>
   <span mat-dialog-title i18n *ngSwitchCase="'add'">Change Collections</span>
 </ng-container>
-<mat-dialog-content>
+<mat-dialog-content style= "max-height: calc(100vh - 310px)">
```

Comment:
@ScottHuangNYU we do not want to implement inline styling. Can you move this to style file.

Reply:
@lmmrssa I am working on it.